packages/tracker/tracker.js

```diff
@@ -441,17 +441,23 @@ Tracker.flush = function (_opts) {
 // so that it is stopped if the current computation is invalidated.
 
 /**
- * @summary Run a function now and rerun it later whenever its dependencies change. Returns a Computation object that can be used to stop or observe the rerunning.
+ * @summary Run a function now and rerun it later whenever its dependencies
+ * change. Returns a Computation object that can be used to stop or observe the
+ * rerunning.
  * @locus Client
- * @param {Function} runFunc The function to run. It receives one argument: the Computation object that will be returned.
+ * @param {Function} runFunc The function to run. It receives one argument: the
+ * Computation object that will be returned.
+ * @param {Function} [onError] Optional. The function to run when an error
+ * happens in the Computation. The only argument it recieves is the Error
+ * thrown. Defaults to the error being logged to the console.
  * @returns {Tracker.Computation}
  */
-Tracker.autorun = function (f) {
+Tracker.autorun = function (f, onError) {
```

Comment:
Since most users of `Tracker.autorun` (a very user-visible API) won't use this optional feature, it should probably be a named option?